### PR TITLE
doc: eks cluster with vpc ref tag

### PR DESCRIPTION
**Pull request by luiscamaral**
### What does this PR do?

Fix reference link on example docs.  (#120)

### Motivation

Broken reference links not pointing to a valid address. (_broken link_)

### More

- [ ] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)
- [ ] Yes, I have added a new example under [examples](https://github.com/aws-observability/terraform-aws-observability-accelerator/tree/main/examples) to support my PR
- [ ] Yes, I have created another PR for add-ons under [add-ons](https://github.com/aws-samples/eks-blueprints-add-ons) repo (if applicable)
- [ ] Yes, I have updated the [docs](https://github.com/aws-observability/terraform-aws-observability-accelerator/tree/main/docs) for this feature
- [ ] Yes, I ran `pre-commit run -a` with this PR

**Note**: Not all the PRs required examples and docs except a new pattern or add-on added.

### For Moderators
- [ ] E2E Test successfully complete before merge?

#### [Add-ons](https://github.com/aws-samples/eks-blueprints-add-ons) project was checked and no references was found

image

### CODE REVIEW

File: examples/existing-cluster-nginx/README.md [hunk 1]
```diff
@@ -46,12 +46,12 @@ Specify the AWS Region where the resources will be deployed. Edit the `terraform
 4. Amazon EKS Cluster
 
 To run this example, you need to provide your EKS cluster name.
-If you don't have a cluster ready, visit [this example](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples/eks-cluster-with-new-vpc)
+If you don't have a cluster ready, visit [this example](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/v4.13.1/examples/eks-cluster-with-new-vpc)
 first to create a new one.
 
 Add your cluster name for `eks_cluster_id="..."` to the `terraform.tfvars` or use an environment variable `export TF_VAR_eks_cluster_id=xxx`.
 
-5. Amazon Managed Service for Prometheus workspace (optional)
```
Comment (bonclay7) [inline examples/existing-cluster-nginx/README.md:54]: Can you keep the number five for this?

**Review by bonclay7 — CHANGES_REQUESTED** [review]
Thanks so much for adressing that, can you please fix the numbers on the nginx doc?

File: examples/existing-cluster-nginx/README.md (hunk 1 above)
Comment (luiscamaral) [reply examples/existing-cluster-nginx/README.md:54]: oops... not sure how it happened.

**Review by bonclay7 — DISMISSED** [review]

**Comment by bonclay7** [comment]
@luiscamaral there's one other place where we have this link as well `docs/helpers/new-eks-cluster.md`. while you're at it could you please fix?

**Comment by luiscamaral** [comment]
Found yet another link missing the version and fixed it

[`./docs/helpers/new-eks-cluster.md:4:...`](https://github.com/aws-observability/terraform-aws-observability-accelerator/tree/v1.6.3/docs/helpers/new-eks-cluster.md)

**Review by bonclay7 — APPROVED** [review]

**Comment by bonclay7** [comment]
Thanks a lot @luiscamaral

**Pull request merged**